code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1237,17 +1237,16 @@
 
 /datum/reagent/stimulum/on_mob_add(mob/living/L)
 	..()
-	L.add_trait(TRAIT_GOTTAGOFAST, id)
+	L.add_trait(TRAIT_STUNIMMUNE, id)
```

Comment:
may I suggest making it halve stuns instead of full immunity?

Reply:
scared?